lib/elixir/lib/calendar/iso.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Calendar.ISO do
```

Comment:
If it's ok with you, I'd like to submit a PR with my multi-calendar implementation I'm working on. It'll probably be a few days before it's ready, but has a full calendar implementation for Gregorian (proleptic) + ISO extensions, Julian (proleptic), and I'm hoping to also have at least Hebrew finished so that we can see how another modern and vastly more complex calendar, fits in to the API. We can't just lean on `:calendar` for Gregorian, because the basic functions required for conversions between calendars must be built up from a shared fixed date system. I can describe more in detail in the PR.